README.md

```diff
@@ -1,9 +1,23 @@
 # OSU Icon Field
+
 Custom Drupal field for osu and fontawesome icons
 
 ## Compile JS
+
 To compile once \
-`npm run build`
+
+```shell
+npm run build
+```
 
 To continuously compile \
-`npm run watch`
+
+```shell
+npm run watch
+```
+
+To Generate a new list of OSU Icons, replace osu-icon-url with the latest url.
+
+```shell
+node js/generate-icon-names.js osu-icon-url
```

Comment:
You can add an npm script in package.json to make this simpler.

`"generate-icons": "node js/generate-icon-names.js"`

Then you just call `npm run generate-icons osu-icon-url`.

Reply:
Good Idea, I've updated it.